models/DataObject/Concrete.php

```diff
@@ -167,15 +167,23 @@ protected function update($isUpdate = null, $params = [])
             }
         }
 
+        $preUpdateEvent = new DataObjectEvent($this, [
```

Comment:
This could be inside the `if` condition, right?

Reply:
No, because then you cannot add validations if there are no exceptions from the default Pimcore validations.

Reply:
Ah, sorry, missed the one crucial line, thanks for clarifying 😊